usage.md

```diff
@@ -0,0 +1,134 @@
+# Usage
+
+## resolve-tags plugin
+
+See the [resolve-tags plug-in guide](https://github.com/grafeas/kritis/blob/master/cmd/kritis/kubectl/plugins/resolve/README.md)
+
+## Resource Reference
+
+Installing Kritis, creates a number of resources in your cluster. Mentioned below are important ones.
```

Comment:
nit: might be worth adding a link to where the charts/hook-images so users have a referenced way to see every resource that will be added.  Not all of the primitives are actually in yaml as well (some are embedded in go) so perhaps having a full list somewhere would be helpful

Reply:
Will do this in a follow-up PR.